dbt_coves/tasks/load/fivetran.py

```diff
@@ -217,18 +236,26 @@ def _get_existent_destination(self, target_group_id):
                 return dest_data
         return {}
 
-    def _load_fivetran_object_secrets(self, obj):
+    def _load_fivetran_local_secrets(self, obj):
         """
         Identify secret files' key:values and replace Fivetran object ones
         """
-        for secret in self.loaded_secrets:
+        for secret in self.local_secrets:
             for fivetran_obj_name, secret_data in secret.items():
                 for destination_data in obj.values():
                     object_details = destination_data["details"]
                     if fivetran_obj_name == object_details["id"]:
                         for k, v in secret_data.items():
                             self._replace_dict_key(object_details, k, v)
 
+    def _load_fivetran_datacoves_secrets(self, object):
```

Comment:
Not familiar with this code so I don't know but... Isn't there a better name than `object` for this?